server/api/getters.py

```diff
@@ -1,49 +1,62 @@
 # all GET request helpers go in here
 from PIL import Image, ImageDraw
+from typing import Any, Dict, Union, List, Tuple
 from .functions import astar
 from flask import send_file
 from flask import jsonify
 from pathlib import Path
 import json
 import io
 import random
+from io import BytesIO
```

Comment:
we don't need this import if you remove the type hint (because that one is unneccesary as well)